src/hyperion/experiment_plans/tests/test_flyscan_xray_centre_plan.py

```diff
@@ -327,6 +329,7 @@ def test_logging_within_plan(
                 "s4_slit_gaps_xgap": 0,
                 "s4_slit_gaps_ygap": 0,
                 "attenuator_actual_transmission": 0,
+                "flux_flux_reading": 10,
```

Comment:
Could: It's annoying that this dictionary is basically the same in 3 places, can you pull it into a constant?